ofl/sahitya/METADATA.pb

```diff
@@ -1,5 +1,5 @@
 name: "Sahitya"
-designer: "Juan Pablo del Peral"
+designer: "Juan Pablo del Peral, Huerta Tipográfica"
```

Comment:
@applecuckoo please remove changes to METADATA.pb in this PR. If the author gives consent to change the credit then it should be made at their demand, and in a separated PR :)

Reply:
Reverted.